core/tests/puppeteer-acceptance-tests/voiceover-tests/play-uploaded-recording.js

```diff
@@ -0,0 +1,23 @@
+const e2eVoiceoverAdmin = require("../utility-functions/voiceoverAdminUtils.js");
+const testConstants = require("../utility-functions/testConstants.js");
+
+
+const homePage = testConstants.Dashboard.MainDashboard;
+const creatorDashboardUrl = testConstants.URLs.CreatorDashboard;
+
+
+async function playUploadedAudioAsVoiceoverAdmin() {
+  const voiceoverAdmin = await new e2eVoiceoverAdmin();
+  await voiceoverAdmin.openBrowser();
+  
+  await voiceoverAdmin.signInWithEmail("testadmin@example.com");
+  await voiceoverAdmin.waitForPageToLoad(homePage);
+  await voiceoverAdmin.goto(creatorDashboardUrl);
+  await voiceoverAdmin.gotoTranslationTabInNewExploration();
+  await voiceoverAdmin.uploadAudioFile('A4.mp3');
```

Comment:
Don't they need to select a language first?

Reply:
by default, the language for voiceovers is English

Reply:
Ah ok, maybe add a comment explaining that.